Mathlib/LinearAlgebra/Matrix/ToLin.lean

```diff
@@ -901,9 +917,7 @@ variable [SMul R S] [IsScalarTower R S M₁] [IsScalarTower R S M₂]
 /-- The natural equivalence between linear endomorphisms of finite free modules and square matrices
 is compatible with the algebra structures. -/
 def algEquivMatrix' [Fintype n] : Module.End R (n → R) ≃ₐ[R] Matrix n n R :=
-  { LinearMap.toMatrix' with
-    map_mul' := LinearMap.toMatrix'_comp
-    commutes' := LinearMap.toMatrix'_algebraMap }
+  LinearMap.toMatrixAlgEquiv'
```

Comment:
This looks like we should deprecate one in favour of the other? Should be in a different PR though.

Reply:
Yep, I figured I'd just leave a breadcrumb here in the meantime.